app/code/Magento/Sales/Model/Order/Config.php

```diff
@@ -117,12 +119,14 @@ public function getStateDefaultStatus($state)
     /**
      * Retrieve status label
      *
-     * @param   string $code
-     * @return  string
+     * @param string $code
+     * @param string|null $forceArea
+     * @return string
+     * @throws LocalizedException
      */
-    public function getStatusLabel($code)
+    public function getStatusLabel($code, $forceArea = null)
```

Comment:
@phoenix128 , adding new parameters to public method is not allowed because this is breaking changes. Please, refer to [Backward compatible development guide](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) section **Adding parameters in public methods** for more information

Reply:
@sidolov , does this even apply for optional parameters?

Reply:
@phoenix128 , yes, this changes may affect 3d party code with the same changes but with different logic